src/Altinn.Apps/AppTemplates/AspNet/App.IntegrationTests/Data/apps/tdd/endring-av-navn/config/applicationmetadata.json

```diff
@@ -20,5 +20,6 @@
       },
       "taskId":  "Task_1"
     }
-  ]
+  ],
+  "autoDeleteOnProcessEnd": false
```

Comment:
Default value for this is false. So, how about we don't define it here and still ensure that autodelete doesnt happen ?

Reply:
Done